lib/pbench/cli/agent/commands/results/__init__.py

```diff
@@ -1,94 +1,173 @@
+"pbench-move-results"
+
 import os
+import click
 import requests
 import shutil
 import sys
 import tempfile
 
-from pbench.agent import PbenchAgentConfig
+from pbench.agent.base import BaseCommand
 from pbench.agent.results import MakeResultTb, CopyResultTb
 from pbench.common.logger import get_pbench_logger
+from pbench.cli.agent import pass_cli_context
+from pbench.cli.agent.options import common_options
 
 
-def move_results(ctx, _user, _prefix, _show_server):
-    config = PbenchAgentConfig(ctx["args"]["config"])
-    logger = get_pbench_logger("pbench", config)
-
-    controller = os.environ.get("_pbench_full_hostname")
-    if not controller:
-        logger.error("Missing controller name (should be 'hostname -f' value)")
-        sys.exit(1)
-
-    results_webserver = config.results.get("webserver")
-    if not results_webserver:
-        logger.error(
-            "No web server host configured from which we can fetch the FQDN of the host to which we copy/move results"
-        )
-        logger.debug("'webserver' variable in 'results' section not set")
-
-    server_rest_url = config.results.get("server_rest_url")
-    response = requests.get(f"{server_rest_url}/host_info")
-    if response.status_code not in [200, 201]:
-        logger.error(
-            "Unable to determine results host info from %s/host_info", server_rest_url
-        )
-        sys.exit(1)
-    if response.text.startswith("MESSAGE"):
-        message = response.text.split("===")[1]
-        logger.info("*** Message from sysadmins of %s:", results_webserver)
-        logger.info("***\n*** %s", message)
-        logger.info("***\n*** No local actions taken.")
-        sys.exit(1)
-    results_path_prefix = response.text.split(":")[1]
-    if not results_path_prefix:
-        logger.error(
-            "fetch results host info did not contain a path prefix: %s", response.text
-        )
-        sys.exit(1)
+class MoveResults(BaseCommand):
+    def __init__(self, context):
+        super().__init__(context)
```

Comment:
The super class init will be called still if this is not present.

Reply:
Confirmed...more or less.  (And, shame on us for doubting it....)

If `MoveResults:__init__()` is not defined, then, when the class is instantiated, the implementation of `__init__()` is _inherited_ from the superclass and it is called instead.

But, yeah, strictly speaking, we don't _need_ this code, although, I don't think it really hurts much to have it.

Reply:
I will just leave it as it is for now